modules/installing-oci-about-assisted-installer.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: CONCEPT
+[id="installing-oci-about-assisted-installer_{context}"]
+= The Assisted Installer and OCI overview
+
+From {product-title} {product-version} and later versions, you can install an {product-title} cluster on {oci-first} virtual machines (VMs) by using the Assisted Installer. You can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments. Both Red Hat and Oracle test, validate, and support running {oci} in an {product-title} cluster on {oci}.
+
+{oci} provides services that can meet your needs for regulatory compliance and performance and cost-effectiveness.
+
+[IMPORTANT]
+====
+To ensure the best performance conditions for your cluster workloads that operate on {oci}, ensure volume performance units (VPUs) for your block volume are sized for your workloads. The following list provides guidance for selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: 100 GB, and 20 to 30 VPUs.
+* Basic environment: 500 GB, and 60 VPUs.
+* Heavy production environment: More than 500 GB, and 100 or more VPUs.
+
+Consider reserving additional VPUs to provide sufficient capacity for updates and scaling activities. For more information about VPUs, see Volume Performance Units in the Oracle documentation.
+====
+
+If you are unfamiliar with the {product-title} Assisted Installer, see the {product-title} documentation section named "Using the Assisted Installer".
```

Comment:
Optional change:
If you are unfamiliar with the OpenShift Container Platform Assisted Installer, see "Using the Assisted Installer" in _Additional Resources_.